grc/core/Messages.py

```diff
@@ -40,7 +40,10 @@ def send(message):
         message: a message string
     """
     for messenger in MESSENGERS_LIST:
-        messenger(_indent + message)
+        try:
+            messenger(_indent + message)
+        except:
+            pass
```

Comment:
I feel like we should be at least logging this? Otherwise, we'll have hard-to-debug problems if for any reason in the future, a crucial messenger permanently fails.

Reply:
You mean, using the regular Python logger instead of the messenger logger? Sure we can add that.